Contributing.md

```diff
@@ -53,21 +56,35 @@ Welcome to the CARMAPlatform contributing guide. There are a few guidelines that
 * When you have finished making your changes, you'll need to push up your changes to a topic branch in your fork of the repository.  Read the article ["Committing and reviewing changes to your project"](https://help.github.com/desktop/guides/contributing-to-projects/committing-and-reviewing-changes-to-your-project/) on GitHub. 
 * With them all committed, push them, this pushes everything in that branch up. 
 
-### Submitting Changes
+#### Submit Changes
 **Let you tell others about changes you've pushed to a GitHub branch.**
 * A Pull Request is the way to notify the project maintainers that you have some work that they should review and add to the project. 
 * You’re requesting that they pull your changes in. Read the article ["Using Pull Requests"](https://help.github.com/articles/using-pull-requests) on GitHub.
 * Then you can go back to your forked branch and issue a pull request to the repository and choose the correct original branch (should be **`develop`**).
 * To create one, go to your fork of the project, click on the Pull Requests tab, and click the big green “New Pull Request” button.
 
-### Merging Changes
+#### Merge Changes
 **We want your help to make CARMAPlatform Project great.**

```

Comment:
I am surprised Taylor doesn't want to change the "spelling" of CARMAPlatform and split it into two words.  Might want to check for other occurrences of this word and double-check with him.